### PR TITLE
Basic CLI for backup and restore

### PR DESCRIPTION
**Pull request by JeffLabonte**

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -1 +1,26 @@
...
+
+SOFTWARE_NAME = 'Linux Migration Assistant - CLI'
+SOFTWARE_DESCRIPTION = """
+This tool is there to help you make the migration of your Linux Machine to your
+new Linux machine. We want to help provide the same functionality that macOS has.
+"""
+
+
+if __name__ == '__main__':
+    parser = ArgumentParser(prog=SOFTWARE_NAME, description=SOFTWARE_DESCRIPTION)
```
Comment (realiceman) [inline src/app.py:14]: c'est pas "argparse.ArgumentParser.." ?

Comment (JeffLabonte) [reply src/app.py:14]: Nope, cause I have imported ArgumentParser from argparse.

See line 2 :)

Comment (realiceman) [reply src/app.py:14]: right !

**Comment by JeffLabonte** [comment]
Thx bug @realiceman

**Pull request merged**